### PR TITLE
Fix Quay image build failure in quay-test repo

### PR DESCRIPTION
**Pull request by xiaozhisean**
update build_root

### CODE REVIEW

**Comment by SeanZhao-redhat** [comment]
/pj-rehearse pull-ci-quay-quay-tests-master-omr-ocp416-images

**Comment by SeanZhao-redhat** [comment]
/pj-rehearse pull-ci-quay-quay-tests-master-omr-ocp416-images

**Comment by LiZhang19817** [comment]
/pj-rehearse pull-ci-quay-quay-tests-master-omr-ocp416-images

File: ci-operator/config/quay/quay-tests/quay-quay-tests-master__omr-ocp416.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ images:
         PACKAGES="openssh-clients httpd-tools nodejs yarn xorg-x11-server-Xvfb gtk2-devel gtk3-devel libnotify-devel GConf2 nss libXScrnSaver alsa-lib skopeo podman" && \
```
Comment (LiZhang19817) [inline ci-operator/config/quay/quay-tests/quay-quay-tests-master__omr-ocp416.yaml:16]: change to use latest NodeJS, https://rpm.nodesource.com/setup_lts.x

**Comment by SeanZhao-redhat** [comment]
/pj-rehearse pull-ci-quay-quay-tests-master-omr-ocp416-images

**Comment by SeanZhao-redhat** [comment]
/pj-rehearse pull-ci-quay-quay-tests-master-omr-ocp416-images

**Pull request merged**